tests/syntax/test_parser.py

```diff
@@ -37,6 +37,26 @@ def test_ego_keyword(self):
             parse_string_helper("ego, x = 10, 20")
 
 
+class TestModel:
+    def test_basic(self):
+        mod = parse_string_helper("model some_model")
+        stmt = mod.body[0]
+        match stmt:
+            case Model("some_model"):
+                assert True
+            case _:
+                assert False
+
+    def test_dotted(self):
+        mod = parse_string_helper("model scenic.simulators.carla")
```

Comment:
Why not have this use `scenic.simulators.carla.model` to check the soft-keywordness of `model`?

Reply:
Not sure why I omitted the most important part. Fixed in 9d130fd7daf0c89e063699c130d40b024f6d5e0e